examples/complete/main.tf

```diff
@@ -169,6 +169,16 @@ locals {
     }
     EOF
   }
+  email_input_transformer = {
```

Comment:
I don't think there is a need in both `order_input_transformer` and `email_input_transformer` since they are almost identical. And we can save space in this large example :)

Reply:
ok, will remove `email_input_transformer` then